README.md

```diff
@@ -31,22 +67,42 @@ but it is optimized for read speed.
   permissions etc.) is provided in a streaming-friendly way.
 
 To efficiently determine which parts of a file changed, the tool uses the same
-CDC-based diffing algorithm as CDC RSync. Changes to Windows files are almost
+CDC-based diffing algorithm as `cdc_rsync`. Changes to Windows files are almost
 immediately reflected on Linux, with a delay of roughly (0.5s + 0.7s x total
 size of changed files in GB).
 
+<p align="center">
+  <img src="docs/cdc_stream_demo.gif" alt="cdc_stream demo" width="688" />
+</p>
+
 The tool does not support writing files back from Linux to Windows; the Linux
 directory is readonly.
 
+The following chart compares times from starting a game to reaching the menu.
+In one case, the game is streamed via `sshfs`, in the other case we use
+`cdc_stream`. Overall, we see a **2x to 5x speedup**.
+
+<p align="center">
+  <img src="docs/cdc_stream_vs_sshfs.png" alt="Comparison of cdc_stream and sshfs" width="752" />
+</p>
+
 # Getting Started
 
-The project has to be built both on Windows and Linux.
+Precompiled binaries can be downloaded from the
+[Github homepage](https://github.com/google/cdc-file-transfer/tags).
+We currently provide Linux binaries compiled on
+[Github's latest Ubuntu](https://github.com/actions/runner-images) version.
+If the binaries work for you, you can skip the following two sections.
+
+Alternatively, the project can be built from source. Some binaries have to be
+built on Windows, some on Linux.
 
 ## Prerequisites
 
-The following steps have to be executed on **both Windows and Linux**.
+To build the tools from source, the following steps have to be executed on
+**both Windows and Linux**.
 
-* Download and install Bazel from https://bazel.build/install.
+* Download and install Bazel from [here](https://bazel.build/install).
```

Comment:
Logs say it's actually 5.3.2. I would rather not add this since it's quickly stale, and it can be changed by Github without us knowing. I'll refer to the workflow logs.

Reply:
Good point about the Bazel version on Github. My thought was that if Bazel releases v6 and it breaks our build, that ppl are not surprised by this. Is there a place that shows the Bazel version which Github is using, so that we can point there?

Reply:
Not that I'm aware of, but the github workflows I'm pointing at show the version in the logs.